R/resolve.R

```diff
@@ -31,17 +31,17 @@ NULL
 #' @export
 min.messydt <- function(..., na.rm = TRUE) {
   x <- list(...)
-  y <- expand(x[[1]])
-  y <- sapply(y, function(x) as.character(min(x, na.rm = na.rm)))
+  y <- sapply(x, function(y) ifelse(is_uncertain(y), expand(y), y))
+  y <- sapply(y, function(x) min(x, na.rm = na.rm))
   y
 }
 
 #' @rdname resolve
 #' @export
 max.messydt <- function(..., na.rm = TRUE) {
   x <- list(...)
-  y <- expand(x[[1]])
-  y <- sapply(y, function(x) as.character(max(x, na.rm = na.rm)))
+  y <- sapply(x, function(y) ifelse(is_uncertain(y), expand(y), y))
```

Comment:
not sure if this is a big issue but I see Codefactor checks failing on this 'sapply'

Reply:
Thank you Jael! I had seen this issue and tried to change, but I am not able to achieve the same returns as with sapply() ... I will keep on looking or an alternative.

Reply:
Solved :)

Reply:
Great solution, thank you @BBieri !